src/canon/Physics.jl

```diff
@@ -233,7 +232,7 @@ end)
 )
 
 @docapode("IceBlockingWater"
-  ,"https://www.google.com"
+  ,""
   ,""
```

Comment:
This model is an approach to modeling interface conditions of an obstruction within a flow developed by J. Fairbanks and L. Morris during the DARPA ASKEM hackathons. It should not be relied on as a realistic model of the physics of obstructions to flows.